schemas/config.json

```diff
@@ -0,0 +1,397 @@
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "metadata": {
+                            "title": "Metadata",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        }
+                    },
+                    "required": [],
+                    "dependencies": {
+                        "relations": [
+                            "database",
+                            "schemas"
+                        ],
+                        "schemas": [
+                            "database"
+                        ]
+                    }
+                },
+                "properties": {
+                    "title": "Properties",
+                    "description": "Generate properties parameters",
+                    "type": "object",
+                    "properties": {
+                        "templates_folder": {
+                            "title": "Templates folder",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "metadata": {
+                            "title": "Metadata",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "update-strategy": {
+                            "title": "Update strategy",
+                            "description": "Action to perform when a property file already exists. Options: update, recreate, fail, ask (per file)",
+                            "type": "string",
+                            "enum": [
+                                "update",
+                                "recreate",
+                                "fail",
+                                "ask"
+                            ]
+                        },
+                        "destination": {
+                            "title": "Destination",
+                            "description": "Where models yml files will be generated\nYou can specify a different path by declaring it explicitly, i.e.: \"models/staging/{{model_file_name}}.yml\"",
+                            "type": "string",
+                            "minLength": 1,
+                            "pattern": ".+({{model_file_name}}.yml)$"
+                        },
+                        "models": {
+                            "title": "Models",
+                            "description": "Model(s) path where 'generate properties' will look for models for generation",
```

Comment:
Filter path used to limit where generate properties will look for model (sql) files.